src/store/reducers/hackathonReducer.js

```diff
@@ -2,7 +2,8 @@ import * as ActionType from "../actions";
 
 const initialState = {
   items: {},
-  assets: []
+  assets: [],
+  hackDetails: null
```

Comment:
we should think about splitting our reducer before it gets too crazy.

https://redux.js.org/api/combinereducers/